src/main/docs/guide/breaks.adoc

```diff
@@ -1,3 +1,9 @@
+=== Micronaut Problem JSON {version}
```

Comment:
Please replace `{version}` with the actual number where this change will take effect.  `{version}` will keep changing.